controls/radgridview/troubleshooting/performance.md

```diff
@@ -41,43 +41,14 @@ You might notice a reduced performance of RadGridView control. Below are listed
             
 * As of __Q2 2014 SP__ release we introduced the option to __turn off generating of automation peers through the new global AutomationMode property of the AutomationManager__. You can check the [UI Automation Support]({%slug common-ui-automation%}) article on how to set the AutomationMode property to Disabled.
 
-* __Alternation__ is a very slow operation. Avoid it unless it is definetely needed. 
+* Applying a __RowStyle__ or an __AlternateRowStyle__ that modifies the ControlTemplate with which the GridViewRow is created will result in a performance hit.
```

Comment:
The phrase "the ControlTemplate with which the GridViewRow is created" sounds too complicated to me. Can't we just use "the default GridViewRow control template"?